packages/swing-store/src/swingStore.js

```diff
@@ -200,6 +200,15 @@ function makeSwingStore(dirPath, forceReset, options = {}) {
     // { verbose: console.log },
   );
```

Comment:
Wondering if we shouldn't take the opportunity to set the timeout at 0 as well, to prevent any busy waits.

```suggestion
  {
    // verbose: console.log,
    timeout: 0,
  });
```

Reply:
I disagree, in my experiments so far, that makes things pretty fragile. I'm ok with the application experiencing jank if we're doing some manual interaction with the DB (something like `sqlite3 swing-store.sqlite 'DELETE FROM historical_transcript_spans'` comes to mind), I'm not so ok with causing it to crash if that operation takes too long. 5s is a probalistic measure, to be sure, but kernel insta-death just feels really risky.